components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1223,7 +1223,24 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
                 context = prebuiltWorkspace;
             }
 
-            await mayStartWorkspacePromise;
+            try {
+                await mayStartWorkspacePromise;
+            } catch (error) {
+                // if the user is not migrated, yet, and the selected organization doesn't have credits, we fall back to what is set on the user preferences
+                // can be deleted when everyone is migrated
+                if (
```

Comment:
Why not making this predictable and always prefer the the "Default Billing Account"? I fear that we'll have a hard time explaining this behavior, even internally.

I understand and am aligned with getting rid of those "two ways of attribution", but I'd like to stay "simple" even for the steps in between.

Reply:
The user settings is only a fallback. So if I'm on org A and start a workspace it is very expected that the workspace will belong to that org. The fall back would only come into play if that org doesn't have credits.

Reply:
Did we communicate it that way? What's user's expectation? IMO so war we communicated "control attribution via Default Billing Account". What are the steps we take to change this? What's explained in the upcoming blog post? :thinking: